service/controller/v12/resource/instance/create.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const (
 	vmssDeploymentName = "cluster-vmss-template"
+	dockerDiskName     = "DockerDisk"
```

Comment:
Move these to `key.DockerVolumeDiskName` and corresponding `key.KubeletVolumeDiskName` as noted below.

Reply:
Done